### PR TITLE
added index in v

### PR DESCRIPTION
**Pull request by vakodrazan**
Hey,

We hope that It is finished. Please have a look.

Peta and Noeline

### CODE REVIEW

**Review by ndorfin — CHANGES_REQUESTED** [review]
This looks great, good job.
But the folder name is still uppercase V.
Please make it lowercase.

**Comment by vakodrazan** [comment]
I think It's fine now, isn't it. Please have a look.

File: v/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# List of artists starting with V
+
+- [Vaiavy chilla](vaiavy-chilla.md)
+- [Vivy](vivy.md)
+- [Voahangy](voangy.md)
+- [Voary](voary.md)
+
+[Go back to the artist index](http://../index.md)
```
Comment (ndorfin) [inline v/index.md]: This link needs to have the `http://` removed as we just want to use relative URLs.

File: v/index.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# List of artists starting with V
+
+- [Vaiavy chilla](vaiavy-chilla.md)
+- [Vivy](vivy.md)
+- [Voahangy](voangy.md)
```
Comment (ndorfin) [inline v/index.md]: This filename doesn't match the artist name, shouldn't it be `voahangy.md`?

**Review by ndorfin — CHANGES_REQUESTED** [review]
Folder name looks good now, thanks!
But I notcied some other erros in your index file:

**Comment by vakodrazan** [comment]
We fixed it but we don't know if it will work.

File: v/index.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# List of artists starting with V
+
+- [Vaiavy chilla](vaiavy-chilla.md)
+- [Vivy](vivy.md)
+- [Voahangy](voahangy.md)
+- [Voary](voary.md)
+
+[Go back to the artist index](http://artist-starting-with-v/index.md)
```
Comment (ndorfin) [inline v/index.md]: This URL should read: `(../index.md)`

**Review by ndorfin — CHANGES_REQUESTED** [review]

**Review by ndorfin — APPROVED** [review]
Well done!

**Pull request merged**